### PR TITLE
Django 1.10 testing + compatability

### PR DESCRIPTION
**Pull request by vegitron**

### CODE REVIEW

File: restclients/templates/proxy/grad/index.html [hunk 1]
```diff
@@ -1,21 +1,21 @@
 <h1>MyGrad Search (Proxy)</h1>
 <ul>
-    {% url 'restclients.views.proxy' 'grad' 'services/students/v1/api/committee.html' as committee_url %}
+    {% url 'restclients_proxy' 'grad' 'services/students/v1/api/committee.html' as committee_url %}
```
Comment (fanglinfang) [inline restclients/templates/proxy/grad/index.html]: Hi Patrick, could you help me understand the effect of this change?

Comment (vegitron) [inline restclients/templates/proxy/grad/index.html]: https://docs.djangoproject.com/en/1.10/releases/1.8/#passing-a-dotted-path-to-reverse-and-url

Comment (fanglinfang) [inline restclients/templates/proxy/grad/index.html]: can the same urlpattern name be used in supporttools?

Comment (vegitron) [inline restclients/templates/proxy/grad/index.html]: @fanglinfang i think if there's a question on the supporttools we should discuss it there, unless it's blocking this pull request

**Pull request merged**